spacer/models/effcientnet_utils.py

```diff
@@ -0,0 +1,298 @@
+"""
```

Comment:
I know these files are borrowed from another repo, but can you do a PEP8 clean-up anyways? Check line length etc.? I have been using 79 character PEP8 line length.

Reply:
sure, I've been using 120 but I will change it to 79 for consistency